cmake/GNU.cmake

```diff
@@ -0,0 +1,64 @@
+set (FOPT0 "-O0")
```

Comment:
This file is not needed.
The last 2 lines are all that are being used.
Please follow one of the other NCEPLIBS- libraries. e.g. nemsio or bacio